File: clients/java/examples/tables-drop-table-name.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // Drop a table
    database.dropTable("**TABLE_NAME**");
  }
}

File: clients/go/examples/tables-drop-table-name.go
package main

import (
	"context"
	"log"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	ctx := context.Background()

	// Get a database
	client := astra.NewClient()

	database := client.Database(
		"**API_ENDPOINT**",
		options.API().SetToken("**APPLICATION_TOKEN**"),
	)

	// Drop a table
	err := database.DropTable(ctx, "**TABLE_NAME**")
	if err != nil {
		log.Fatal(err)
	}
}
